# a msg (signal) to tell the camera controller to take a shot
Header Header
bool if_take